android/engine/src/main/java/org/smartregister/fhircore/engine/navigation/RegisterBottomSheetItem.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.navigation
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class RegisterBottomSheetItem(
+  val id: String,
+  val display: String,
+  var showCount: Boolean = false
+)
```

Comment:
I do not think this data class is necessary. All the properties are available in `NavigationMenuConfig` model.

Reply:
The register items will also have actions that can be triggered for instance when the item is clicked.

Reply:
Action is also provided via `NavigationMenuConfig` model.